src/helpers/_font-faces.scss

```diff
@@ -21,47 +21,26 @@
 
       @font-face {
```

Comment:
I believe that we don't need to have the old fonts around here since GOVUK_Template will supply these?

This font-face is already automatically excluded when certain compat flags are present, something to double check.

Reply:
The old fonts are already gone, unless you mean get rid of the .eot versions etc?

Reply:
When legacy mode is enabled, GOV.UK Frontend font-face declarations are not output and GOV.UK Frontend relies on GOVUK_Template being loaded alongside.

So this means we don't need to keep the old fonts around in GOV.UK Frontend, but we should check this.